ExperimentTwitter/TweetsTableViewController.m

```diff
@@ -0,0 +1,174 @@
+//
+//  TweetsViewController.m
+//  ExperimentTwitter
+//
+//  Created by ayur.j on 25/08/17.
+//  Copyright © 2017 ayur.j. All rights reserved.
+//
+
+#import "TweetsTableViewController.h"
+#import "PostTweetViewController.h"
+#import "CustomTweetCell.h"
+#import "TwitterClient.h"
+#import "Tweet+Twitter.h"
+#import "TweetsSearchViewController.h"
+
+@interface TweetsTableViewController ()
+@property (strong, nonatomic) NSMutableArray *tweets;
+@property (strong, nonatomic) UIRefreshControl *refreshControl;
+@property (nonatomic) BOOL loadMoreData;
+@end
+
+@implementation TweetsTableViewController
+
+- (id) initWithTweets:(NSArray *)tweets {
+    self = [super init];
+    self.tweets = [NSMutableArray arrayWithArray:tweets];
+    return self;
+}
+
+@synthesize refreshControl;
```

Comment:
Synthesize is not necessarily required

Reply:
Compiler says auto property synthesis will not synthesize 'refreshControl', it will be implemented by its superclass, use @dynamic to acknowledge intention.